airflow-core/src/airflow/ui/public/i18n/locales/en/admin.json

```diff
@@ -39,7 +39,12 @@
       "selectConnectionType": "Select Connection Type",
       "standardFields": "Standard Fields"
     },
-    "noRowMessage": "No connections found",
+    "nothingFound": {
```

Comment:
Is it ok to ignore other locales or do I need to do something special in there?

Reply:
Would be cool to transfer the keys over where possible

Reply:
Meaning: I'd propose to keep the existing text, then less translation rework. `Nothing found!` --> `No connections found`

Reply:
English is in all languages used as fallback. But a partial translation is better than none. So if you copy the single existing key over this is sufficient and the rest is just pulled from English until translated.

Reply:
done